qa/TL0_multigpu/test_body.sh

```diff
@@ -44,6 +44,10 @@ test_pytorch() {
     ${python_invoke_test} --attr 'multigpu' test_external_source_pytorch_gpu.py
 }
 
+test_jax() {
+    ${python_new_invoke_test} -s . test_jax_integration_multigpu
```

Comment:
I wonder if any other test script fill catch it and run using the old nose?

Reply:
No, it will not as far as I can tell. We have this merged in another PR and it looks like everything worked fine. This follows example of other frameworks.